classes/cargo.bbclass

```diff
@@ -114,8 +116,8 @@ cargo_do_compile() {
     bbnote "rustc --version" `rustc --version`
     bbnote "which cargo:" `which cargo`
     bbnote "cargo --version" `cargo --version`
-    bbnote cargo build ${CARGO_BUILD_FLAGS}
-    cargo build ${CARGO_BUILD_FLAGS}
+    bbnote ${RUSTFLAGS} cargo build ${CARGO_BUILD_FLAGS}
+    RUSTFLAGS="${RUSTFLAGS}" cargo build ${CARGO_BUILD_FLAGS}
```

Comment:
I think it would make sense to remove the RUSTFLAGS assignment at the top-level and to change this to `export RUSTFLAGS=...` in do_compile as is done with the other exported environment variables for consistency.

Note that this can also be done via a `do_compile_prepend` in a bbappend.  I actually believe I am doing this for at least one project I maintain.  That being said, I think this is a fine addition and am happy to include with the suggested change.